src/applications/appeals/995/components/AddIssue.jsx

```diff
@@ -181,6 +181,8 @@ const AddIssue = ({ data, goToPath, setFormData, testingIndex }) => {
           onInput={handlers.onIssueNameChange}
           onBlur={handlers.onInputBlur}
           error={((submitted || inputDirty) && showIssueNameError) || null}
+          message-aria-describedby={content.name.hintText}
```

Comment:
why do we have two here?
`message-aria-describedby` and `messageAriaDescribedby`

Reply:
LOL I was trying to figure out which one the React binding would accept. It work, so it's one of these two - I'll remove the one that doesn't work. Thanks!